Prime Checker/prime checker.cpp

```diff
@@ -5,18 +5,18 @@ int prime(int n)
 {
   	if (n == 1) 
 	{
-    return 0;
+    		return 0;
   	}
   	else 
 	{
-    	for (int i = 2; i < n; i++) 
+    		for (int i = 2; i < n; i++) 
 		{
-      		if (n % i == 0)
-			{
-        		return 0;
-      		}
-    	}
-    	return 1;
+      			if (n % i == 0)
+				{
+        				return 0;
+      				}

```

Comment:
I have made the changes in the indentation. I don't know why this one is showing an extra tab here but there is no such space when the file is opened.